examples/stress_tests/time_smoothness.rs

```diff
@@ -0,0 +1,235 @@
+        .insert_resource(MoveStyle::WholePixel)
+        .add_system(change_settings)
+        .add_system(animate_runner)
+        .add_system(scroll_background)
+        .run();
+}
+
+fn setup(
+    mut commands: Commands,
+    asset_server: Res<AssetServer>,
+    mut texture_atlases: ResMut<Assets<TextureAtlas>>,
+) {
+    // Locate the center(ish) of the background conveyer belt, so we can
+    // position the player and camera there.
+    let bg_center = Vec2::new(BG_WIDTH * BG_TILES as f32 / 2.0, BG_HEIGHT / 2.0).round();
+
+    // Set up camera
+    let mut camera_bundle = OrthographicCameraBundle::new_2d();
+    camera_bundle.orthographic_projection.scale = 1.0 / 4.0;
+    camera_bundle.transform.translation += bg_center.extend(0.0);
+    commands.spawn_bundle(camera_bundle);
+    commands.spawn_bundle(UiCameraBundle::default());
+
+    // Set up animated player sprite
+    let runner_texture: Handle<Image> =
+        asset_server.load("textures/rpg/chars/mani/mani-idle-run.png");
+    let mut runner_atlas = TextureAtlas::from_grid(runner_texture, Vec2::new(24.0, 24.0), 7, 1);
+    // Drop the first (idle) frame so we just have the run frames.
+    runner_atlas.textures = runner_atlas.textures[1..].into();
+    let runner_handle = texture_atlases.add(runner_atlas);
+    // Offset by half our size to find where we should place our bottom left corner.
+    let runner_location = bg_center - Vec2::new(12.0, 12.0);
+    commands
+        .spawn_bundle(SpriteSheetBundle {
+            texture_atlas: runner_handle,
+            sprite: TextureAtlasSprite {
+                anchor: bevy::sprite::Anchor::BottomLeft,
+                ..Default::default()
+            },
+            transform: Transform::from_translation(runner_location.extend(3.0)),
+            ..Default::default()
+        })
+        .insert(Player)
+        .insert(AnimationTimer {
+            timer: Timer::from_seconds(0.1, true),
+        });
+
+    // Set up scrolling background, using a conveyor belt of three long sprites
+    let background_texture: Handle<Image> = asset_server.load("branding/banner.png");
+    for i in 0..BG_TILES {
+        commands
+            .spawn_bundle(SpriteBundle {
+                sprite: Sprite {
+                    anchor: bevy::sprite::Anchor::BottomLeft,
+                    ..Default::default()
+                },
+                transform: Transform::from_translation(Vec3::new(
+                    i as f32 * BG_WIDTH + 1.0,
+                    0.0,
+                    0.0,
+                )),
+                texture: background_texture.clone(),
+                ..Default::default()
+            })
+            .insert(Background);
+    }
+}
+
+fn change_settings(
+    input: Res<Input<KeyCode>>,
+    mut windows: ResMut<Windows>,
+    mut time_style: ResMut<TimeStyle>,
+    mut move_style: ResMut<MoveStyle>,
+) {
+    let window = windows.primary_mut();
+    if input.just_pressed(KeyCode::P) {
+        // P: cycle PresentMode.
```

Comment:
These controls should be mentioned at the start of the example, in the module style comment.